### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by avoinea** [amalfra/docker-hub#2]
It would be nice to align the commands with `docker-compose` ones. We can also inspire from `rancher cli`

- [x] 1. For example, instead of providing the `--orgname` each and every time, we can do:

        $ docker-hub config
        Url: DOCKER_HUB_API_ENDPOINT
        Organization Name: ...
        Items per page: ...

- [x] 2. Rename `repos` to `images` (after all, they are images)

        $ docker-compose images
        $ docker-hub images

- [x] 3. Rename `queue` to `ps`

        $ docker-hub ps

- [x] 4. Move global options before commands and provide a short version:

        $ docker-hub --orgname docker images
        $ docker-hub -o docker images

**Comment by amalfra** [comment]
@avoinea Thanks for the valuable suggestions. 
1. I think this is great feature! Users can additionally pass ```--orgname``` as it is now which will override config. we will show required error only if user hasn't configured using ```docker-hub config``` command and orgname is passed.
2. Isn't it better to have ut as repos only as docker hub ui has repos? I think for docker compose it makes sense because that command shows docker images used for creating running containers. On docker hub on the other hand it's a repo for storing your image, only that image name and repo name are same
3. I think the current PR already has a discussion on the command name
4. It seems like ```docker-hub --orgname myorg --reponame myrepo  tags``` already works. Supporting short version is definitely a good feature

**Comment by avoinea** [comment]
I think you're right about `2`

Also, maybe we should consider removing the table borders (even if I like them) for a cleaner interface and also to be able to pipe other parsing tools (like, I don't know,  `awk`, `grep`, `wc`, etc.).
```
$ docker-compose ps | awk '{print $1}'

vs

$ docker-hub --orgname plone --reponame plone builds | awk '{print $1}'
```

**Comment by amalfra** [comment]
Yeah, output format supporting pipe parsing is great idea. There is already support for 2 types of output formats(using --format argument):
* table
* JSON

maybe we should add new format which can be parsed by tools like awk, grep, wc etc?

**Comment by amalfra** [comment]
All improvements are included in v2.0. @avoinea feel free to let me know if any more improvements are required.
